components/alert/AlertCharacterMetrics.vue

```diff
@@ -10,8 +10,9 @@
       <h1>Loading...</h1>
     </div>
     <div v-if="loaded" class="grid grid-cols-12">
-      <div class="col-span-12 mb-2 flex justify-center">
-        <div class="pr-2 py-2">Player Counts:</div>
+    
+      <div class="col-start-1 col-span-full mb-2 flex flex-wrap justify-center ">
```

Comment:
Not sure if `col-start-1` is required if you're also using `col-span-full`, think it's redundant. ALso note extra space on the end